bazel/repository.bzl

```diff
@@ -50,6 +50,17 @@ def opentelemetry_cpp_deps():
         ],
     )
 
+    maybe(
+        http_archive,
+        name = "com_google_absl_latest11",
+        sha256 = "dcf71b9cba8dc0ca9940c4b316a0c796be8fab42b070bb6b7cab62b48f0e66c4",
+        strip_prefix = "abseil-cpp-20211102.0",
+        urls = [
+            "https://storage.googleapis.com/grpc-bazel-mirror/github.com/abseil/abseil-cpp/archive/20211102.0.tar.gz",
+            "https://github.com/abseil/abseil-cpp/archive/20211102.0.tar.gz",
```

Comment:
repository.bzl is already using:
        strip_prefix = "abseil-cpp-20220623.1",

Per the abseil release notes:
https://github.com/abseil/abseil-cpp/releases/tag/20220623.1
this is the latest abseil version using C++11.

Why do we need to use an older version ?

Reply:
The latest gRPC can not work with a old abseil-cpp , I think we can use a old gRPC and abseil-cpp with legacy toolchain and the latest version with modern toolchain.

Reply:
So, grpc-1.46.4 is the last gRPC release supporting C++11, which is dependent on abseil-cpp-20211102.0.

Reply:
gRPC 1.46.4 is the last with C++11 support which has dependency on abseil-cpp-20211102.0. But it seems it works with the latest abseil as well:
https://github.com/open-telemetry/opentelemetry-cpp/actions/runs/3516378955/jobs/5892867609

I'm going to remove it.